### DISCUSSION TITLE
How can I redirect user on connection error

### DISCUSSION THREAD

**Discussion by danilopinotti** [Hieromon/AutoConnect#429] [How can I?] [answered]
How can I redirect user to "Configure New AP" page when he inserts a wrong network password?

Today, he is redirected to an error page showing the error code, e.g DISCONNECTED (6)

**Comment by Hieromon** [comment, marked as answer]
AutoConnect does not have the options you want. Maybe you can't implement it. In the current AutoConnect program structure, there is no way to properly detect `WIFI_REASON_AUTH_FAIL` from the esp32 arduino core. (Probably not possible with esp32 arduino core.)
Refs: https://www.esp32.com/viewtopic.php?t=2955
